spec/models/task_spec.rb

```diff
@@ -29,6 +30,11 @@
       expect(task).not_to be_valid
     end
 
+    it "is not valid without existing project" do
+      task = FactoryGirl.build(:task)
+      task.save!(validate: false)
```

Comment:
depends on the point of view, less code is more readable for me :trollface: however I can change it, no problem;)